hub/policy/src/integration-test/java/uk/gov/ida/integrationtest/hub/policy/apprule/SessionResourceWithRedisIntegrationTest.java

```diff
@@ -60,6 +60,7 @@
     private static final boolean REGISTERING = true;
     private static final boolean SIGNING_IN = !REGISTERING;
     private static final LevelOfAssurance REQUESTED_LOA = LevelOfAssurance.LEVEL_2;
+    private static final String abTestVariant = null;
```

Comment:
I feel like in these cases it would have been better to stick to the existing pattern i.e. just pass "this-is-a-test-variant-specifier" or something, rather than have a pointless `null` kicking around.